shared/src/bundles/users/types/user-sign-up-response-dto.type.ts

```diff
@@ -1,6 +1,7 @@
 type UserSignUpResponseDto = {
     id: number;
     email: string;
+    token?: string;
```

Comment:
Why it is not required here but required in sign-in?

Reply:
Because UserSignUpResponseDto is also used by create() method in user.service before the creation of the token by signUp() in auth.service.ts. 
Is there a better way to do it?